### PR TITLE
WIP: Go agent rh7

### PR DESCRIPTION
**Pull request by stirlingv**

### CODE REVIEW

File: deploy/roles/gocd/meta/main.yaml [hunk 1]
```diff
@@ -1,8 +1,7 @@
 ---
 dependencies:
   - {role: python, when: "install_python and custom_repo"}
-  #- {role: python-build, when: "not custom_repo"}
-  - {role: python27-scl, when: "not custom_repo"}
+  - {role: python-build, when: "not custom_repo"}
```
Comment (tanderegg) [inline deploy/roles/gocd/meta/main.yaml]: @stirlingv We don't want to undo this change, can you update your code to keep python27-scl, and remove python-build?

File: deploy/roles/gocd/tasks/main.yaml [hunk 2]
```diff
@@ -58,36 +57,41 @@
...
     state: started
+  tags: gocd
 
 - name: Set recursive admin permissions on log folders
   acl:
     path: "{{ item }}"
     entity: "{{ restricted_group_name }}"
     etype: group
-    permissions: rX
+    permissions: rx
```
Comment (tanderegg) [inline deploy/roles/gocd/tasks/main.yaml]: @stirlingv This shoudl be `rX` I'm pretty sure, the capital X makes sure that only folders are set as executable, not files, which is what we want.

File: deploy/roles/gocd/tasks/main.yaml [hunk 3]
```diff
@@ -58,36 +57,41 @@
...
   with_items: "{{ gocd_log_file_locations }}"
+  tags: gocd
 
 - name: Set default admin permissions on log folders
   acl:
     path: "{{ item }}"
     entity: "{{ restricted_group_name }}"
     etype: group
-    permissions: rX
+    permissions: rx
```
Comment (tanderegg) [inline deploy/roles/gocd/tasks/main.yaml]: @stirlingv see above

**Review by tanderegg — CHANGES_REQUESTED** [review]
@stirlingv a few small changes, thanks Stirling!

File: deploy/roles/gocd/meta/main.yaml (hunk 1 above)
Comment (stirlingv) [reply deploy/roles/gocd/meta/main.yaml]: Fixed! henceforth indicated by "👍"

File: deploy/roles/gocd/tasks/main.yaml (hunk 2 above)
Comment (stirlingv) [reply deploy/roles/gocd/tasks/main.yaml]: 👍

File: deploy/roles/gocd/tasks/main.yaml (hunk 3 above)
Comment (stirlingv) [reply deploy/roles/gocd/tasks/main.yaml]: 👍

**Comment by stirlingv** [comment]
@tanderegg Should be good to go! :-)

**Review by tanderegg — APPROVED** [review]

**Review by tanderegg — APPROVED** [review]
